docs/releases/2.7.3.rst

```diff
@@ -0,0 +1,114 @@
+=====================================
+Welcome to the new Pootle 2.7.3 final
+=====================================
+
+*Released on 20 November 2015*
+
+Bugfix release for 2.7.2.
+
+
+Changes in Requirements
+=======================
+- Django >= 1.7.10, < 1.8
+- Translate Toolkit >= 1.13.0
+- Python >= 2.7, < 3.0
+- Redis >= 2.8.4
+- Django transaction hooks
+- Unix-based operating system.
+
+
+Major Changes
+=============
+
+- Pulled latest translations.
+- MyISAM support has been dropped.
+- The editor for static pages now highlights the content's markup and displays
+  a live preview.
+- Added support for Elasticsearch-based external Translation Memory servers.
+- Cleaned up connection logic and added checks for misconfigured Translation
+  Memory servers.
+
+Below we provide much more detail. These are by no means exhaustive, view the
+`git log <https://github.com/translate/pootle/compare/2.7.2...2.7.3>`_ for
+complete information.
+
+
+Details of changes
+==================
+
+- Pulled latest translations.
```

Comment:
I think lets just take that as a given unless maybe we want to brag about new localisations?